lib/liquid/block_body.rb

```diff
@@ -154,14 +154,25 @@ def render_to_output_buffer(context, output)
     private
 
     def render_node(context, output, node)
-      node.render_to_output_buffer(context, output)
+      if node.disabled?(context)
+        output << node.disabled_error_message
+        return
+      end
+      disable_tags(context, node.disabled_tags) do
+        node.render_to_output_buffer(context, output)
+      end
     rescue UndefinedVariable, UndefinedDropMethod, UndefinedFilter => e
       context.handle_error(e, node.line_number)
     rescue ::StandardError => e
       line_number = node.is_a?(String) ? nil : node.line_number
       output << context.handle_error(e, line_number)
     end
 
+    def disable_tags(context, tags, &block)
+      return yield unless tags
```

Comment:
Should this be `if tags.empty?`? (since the default is `[]`, which is truthy)

Reply:
👍 was handling nils before, it's just a shortcut to avoid running extra logic.